extensions/quartz/runtime/src/main/java/io/quarkus/quartz/runtime/QuartzExtensionPointConfig.java

```diff
@@ -20,4 +21,31 @@
     @ConfigItem
     @ConfigDocMapKey("property-name")
     public Map<String, String> properties;
+
+    @ConfigGroup
+    static public class QuartzExtensionPointOptionalConfig {
+        /**
+         * Class name for the configuration.
+         */
+        @ConfigItem(name = "class")
+        public Optional<String> clazz;
+
+        /**
+         * The properties passed to the class.
```

Comment:
It will be good to show an example of how these properties will be passed on. 
Is it a full qualified properties name of a subpart of it.. those are the questions a user might be asking

Reply:
I add comments to field
```java
protected QuartzExtensionPointOptionalConfig customStore
```